src/System Application/App/Performance Profiler/permissions/PerformanceProfilerObjects.PermissionSet.al

```diff
@@ -5,11 +5,19 @@
 
 namespace System.Tooling;
 
+using System.PerformanceProfile;
+
 permissionset 1921 "Performance Profiler - Objects"
 {
     Access = Internal;
     Assignable = false;
 
     Permissions = codeunit "Sampling Performance Profiler" = X,
-                  page "Performance Profiler" = X;
+                  codeunit "Scheduled Perf. Profiler" = X,
+                  page "Performance Profiler" = X,
+                  page "Perf. Profiler Schedules List" = X,
+                  page "Perf. Profiler Schedule Card" = X,
+                  page "Performance Profile List" = X,
+                  tabledata "Performance Profile Scheduler" = Rimd,
```

Comment:
The tabledata should be in another permissionset. view and edit. Edit includes view and view includes this.